src/NetAnalyzers/UnitTests/Microsoft.CodeQuality.Analyzers/ApiDesignGuidelines/TypesThatOwnDisposableFieldsShouldBeDisposableTests.cs

```diff
@@ -44,25 +43,67 @@ public class NoDisposeClass
         [Fact]
         public async Task CA1001CSharpTestWithFieldInitAndNoDisposeMethodAsync()
         {
-            await VerifyCS.VerifyAnalyzerAsync(@"
+            await VerifyCS.VerifyCodeFixAsync(@"
 using System.IO;
 
-    public class NoDisposeClass
+    public class [|NoDisposeClass|]
     {
         FileStream newFile1, newFile2 = new FileStream(""data.txt"", FileMode.Append);
     }
```

Comment:
```suggestion
public class [|NoDisposeClass|]
{
    FileStream newFile1, newFile2 = new FileStream(""data.txt"", FileMode.Append);
}
```

Reply:
Up to you, if you want to fix these indents or not (a few more tests have the same issue).